pkg/services/user/userimpl/store.go

```diff
@@ -107,6 +109,24 @@ func (ss *sqlStore) GetByID(ctx context.Context, userID int64) (*user.User, erro
 	return &usr, err
 }
 
+func (ss *sqlStore) GetByUID(ctx context.Context, orgId int64, uid string) (*user.User, error) {
+	var usr user.User
+
+	err := ss.db.WithDbSession(ctx, func(sess *db.Session) error {
+		has, err := sess.Table("user").
+			Where("org_id = ? AND uid = ?", orgId, uid).
```

Comment:
a is_service_account filter should be added here

Reply:
Rather than add parameters here -- that would change the existing apis used everywhere, i'll check the results in the apiservers